config/initializers/filter_parameter_logging.rb

```diff
@@ -3,4 +3,6 @@
 # Be sure to restart your server when you modify this file.
 
 # Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += [:password]
+Rails.application.config.filter_parameters += [
+  :passw, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn
```

Comment:
is `passw` a type here? I can't find this change in the 6.1 notes

Reply:
It's correct.  This file was created by the rails `app:update` rake task.  You can see an example of `passw` here:

https://github.com/rails/rails/blob/main/railties/lib/rails/generators/rails/app/templates/config/initializers/filter_parameter_logging.rb.tt